Halforbit.DataStores/Extensions/EnumerableExtensions.cs

```diff
@@ -91,5 +93,56 @@ IEnumerable<TResult> Batch(int sz)
                 }
             }
         }
+    
+        
+        internal static async Task<IReadOnlyList<TOut>> ParallelSelectAsync<TIn, TOut>(this IEnumerable<TIn> inputs, Func<TIn, Task<TOut>> mutator, CancellationTokenSource cancellationTokenSource = null, int maxPending = int.MinValue)
```

Comment:
Is there a reason these can't be simply `SelectAsync` and `ForEachAsync`?

Reply:
Nope I just straight stole them from our PC implementations. Renamed in deac7feedd35f2add9219874e4cf76b76ce4e3ef